pubsub/aws/snssqs/snssqs_integ_test.go

```diff
@@ -0,0 +1,310 @@
+// ------------------------------------------------------------
```

Comment:
Move these to the e2e folder until we have a standardized way to write integration tests. See this as example: https://github.com/dapr/components-contrib/tree/master/tests/e2e/bindings/zeebe

Reply:
this is WIP and lots of work. So I think I'll open another PR for that. @artursouza wdyt? I've implemented the suggested changes (thanks, good ones!). I'm going to trigger a new build as it seem to glitch. please re-review

Reply:
I am OK for the E2E test to be in a separate PR. Once we have integration tests, it can land there too.